### ISSUE TITLE
Listbox: Active State does not have 3:1 contrast with adjacent background

### ISSUE THREAD

**Issue by scurker** [dequelabs/cauldron#1214]
Auditor Issue: https://axeauditor.dequecloud.com/test-run/87ff92fc-25f1-11ee-a855-1b2298d1b894/issue/8f22b738-432b-11ee-b594-0fb54ec2eb3e?activeTab=dt-issue&page=0&sortField=ordinal&sortDir=asc&row=3&issuesCount=&issuesCount=&filter=

## Description

The state of an ACTIVE user interface component does not have at least a 3 to 1 luminosity contrast ratio with either the inner or outer adjacent background.

## Element description: 

- Listbox Options
- Disabled Options
- Grouped Options
- Numbers that Cycle
- Controlled Listbox
- Uncontrolled Listbox

Element location: Below the "Examples" heading

### Light theme:

Selected state color code: `#E0E0E0`
Unselected state color code: `#FFFFFF`
Contrast ratio: `1.3:1`

### Dark theme:

Selected state color code: `#02101A`
Unselected state color code: `#283640`
Contrast ratio: `1.5:1`

## Dev Notes

For dark mode, change the background color to `accent-light` and the text color to `accent-medium`.

For light mode, change the background color to `gray-50 / #757575` and text color to `#fff`.
